omod-2.0/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs2_0/AdministrationLinksResource2_0.java

```diff
@@ -45,11 +45,7 @@
 
 	private static final String LINKS = "administrationLinks";
 
-	private ModuleFactoryWrapper moduleFactoryWrapper = new ModuleFactoryWrapper();
-
-	public void setModuleFactoryWrapper(ModuleFactoryWrapper moduleFactoryWrapper) {
-		this.moduleFactoryWrapper = moduleFactoryWrapper;
-	}
+	private static ModuleFactoryWrapper moduleFactoryWrapper = new ModuleFactoryWrapper();
```

Comment:
Why change it to static?

Reply:
I thought that the Whitebox.setInternalState works only on static fields. Changed here: https://github.com/openmrs/openmrs-module-webservices.rest/pull/484/commits/6f97d13cc3bddb3677cd2970b8d51fa7570b5b97